features/scroll-elements.yml

```
@@ -0,0 +1,43 @@
+name: Scroll methods on elements
+description: Scrolling methods such as `scroll`, `scrollBy` and `scrollTo` change the scroll position of an element. Similar to setting th `scrollTop` and `scrollLeft` properties, but also allows options to be passed to define the scroll behavior.
```

```suggestion
description: Scrolling methods such as `scroll()`, `scrollBy()`, and `scrollTo()` change the scroll position of overflow content within an element. Similar to setting `scrollTop` and `scrollLeft` properties, but also allows options defining whether the scroll should animate smoothly or jump.
```

I don't love the "overflow content" portion, but think it could be useful to specify you're calling it on the container that has overflow, not on an element within the scroll container.

Since `scrollTo` is just an alias for `scroll` (I think? Is there a difference?) do we need to mention both?

---

I cannot find a difference, looks like an alias, will update to note that.